From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Wed, 15 Nov 2023 21:53:47 +0100
Subject: build: get rid of CppHeap for now

yolo we need to check how much dies in test

diff --git a/src/env.cc b/src/env.cc
index 1d8df40c3446ac9c72c5f0ad24edf0b8a96cf16b..7fa759042407a1bf6ebfae2f59b0c5a849cca833 100644
--- a/src/env.cc
+++ b/src/env.cc
@@ -543,13 +543,13 @@ IsolateData::IsolateData(Isolate* isolate,
     // safe guard against this.
     DCHECK_NE(descriptor.wrappable_type_index, BaseObject::kSlot);
   } else {
-    cpp_heap_ = CppHeap::Create(
-        platform,
-        CppHeapCreateParams{
-            {},
-            WrapperDescriptor(
-                BaseObject::kEmbedderType, BaseObject::kSlot, cppgc_id)});
-    isolate->AttachCppHeap(cpp_heap_.get());
+    // cpp_heap_ = CppHeap::Create(
+    //     platform,
+    //     CppHeapCreateParams{
+    //         {},
+    //         WrapperDescriptor(
+    //             BaseObject::kEmbedderType, BaseObject::kSlot, cppgc_id)});
+    // isolate->AttachCppHeap(cpp_heap_.get());
   }
   // We do not care about overflow since we just want this to be different
   // from the cppgc id.
